client/hotrod/src/main/java/org/infinispan/hotrod/configuration/HotRodConfigurationBuilder.java

```diff
@@ -276,6 +277,7 @@ public HotRodConfigurationBuilder maxRetries(int maxRetries) {
    @Override
    public HotRodConfigurationBuilder addJavaSerialAllowList(String... regEx) {
       this.allowListRegExs.addAll(Arrays.asList(regEx));
+      attributes.attribute(ALLOW_LIST).set(allowListRegExs.toArray(new String[0]));
```

Comment:
So this is a separate bug? We should probably log a separate JIRA for this.

Reply:
Yes. There are a few other small things I've changed too. I can open a JIRA for them.

Reply:
Created and added a commit.